setup.py

```diff
@@ -75,7 +75,7 @@ def package_files(prefix: str, directory: str, extension: str):
 install_requires = [
     'pyyaml>=6.0,<7',
     'tqdm>=4.62.3,<5',
-    'torchmetrics>=0.7.0,<0.10.0',
+    'torchmetrics>=0.10.0,<0.12.0',
```

Comment:
for posterity's sake, could you prove that the code in this PR works with both 0.10 and 0.11 and put it in the PR description?